docs/install/sm.md

```diff
@@ -1 +1,94 @@
-Two sections (SM on CF and SM on K8S)
\ No newline at end of file
+# Run Service Manager
+
+## Prerequisites
+
+* You need to have an OAuth server to be used by the Service Manager. In CF this could be the CF UAA.
```

Comment:
Can we add a short note that an easy way to get one is with Keycloak? With docker-compose for example - https://github.com/jboss-dockerfiles/keycloak/tree/master/docker-compose-examples you need to define only 1 client in the realm that can be imported from the given json configuration and then change the issuer_url to `http://localhost:8080/auth/realms/master` where `localhost:8080` is the URL of keycloak
[sm.json.txt](https://github.com/Peripli/service-manager/files/2459033/sm.json.txt)

Reply:
Also, this oauth server must support [OpenID Connect Discovery](https://openid.net/specs/openid-connect-discovery-1_0.html)

Reply:
> Can we add a short note that an easy way to get one is with Keycloak? With docker-compose for example - https://github.com/jboss-dockerfiles/keycloak/tree/master/docker-compose-examples you need to define only 1 client in the realm that can be imported from the given json configuration and then change the issuer_url to `http://localhost:8080/auth/realms/master` where `localhost:8080` is the URL of keycloak
> [sm.json.txt](https://github.com/Peripli/service-manager/files/2459033/sm.json.txt)

Isn't it better for this to be in the devguide?